var/spack/repos/builtin/packages/font-util/package.py

```diff
@@ -70,12 +71,19 @@ class FontUtil(AutotoolsPackage, XorgPackage):
         f = f_r[0]
         resource(name=f, url=font_baseurl + f + '-' + f_r[1] + '.tar.gz',
                  sha256=f_r[2], destination=f, when='fonts=' + f)
+
+        conflicts('fonts=font-bh-tff', when='platform=cray')
+        conflicts('fonts=font-bh-tff', when='arch=linux-rhel7-broadwell')
```

Comment:
```suggestion
        conflicts('fonts=font-bh-ttf', when='platform=cray')
        conflicts('fonts=font-bh-ttf', when='arch=linux-rhel7-broadwell')
```
This will fix the failing unit tests.